README.md

```diff
@@ -10,6 +10,7 @@ The directions below explain how to build and deploy the plugins. For more infor
 1. AI21 LLM: Uses AI21's Jurassic model API - requires an AI21 account with an API Key
 2. Anthropic LLM: Uses Anthropic's Claude model API - requires an Anthropic account with an API Key
 3. Amazon Bedrock Embeddings and LLM: Uses Amazon Bedrock service API (preview) - requires access to Amazon Bedrock service (currently in private preview)
+4. Llama 2 13b Chat LLM: Uses Llama 2 13b Chat model - requires Llama-2-chat model to be deployed via SageMaker JumpStart. 
```

Comment:
Add a reference link to point users to guide on how to install the model in SageMaker JumpStart

Reply:
Addressed in the 2nd commit.